### PR TITLE
Fixed issues with Claude Human: Assistant: prompt 

### PR DESCRIPTION
**Pull request by mikejgillespie**
*Issue #, if available:*

*Description of changes:*
Put in explicit prompt templates for Claude calls to ensure the proper prompt format is used.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: 04_Chatbot/00_Chatbot_Claude.ipynb [hunk 1]
```diff
@@ -511,9 +471,6 @@
     "from langchain.indexes.vectorstore import VectorStoreIndexWrapper\n",
     "from langchain.vectorstores import FAISS\n",
     "\n",
-    "s3_path = f\"s3://jumpstart-cache-prod-us-east-2/training-datasets/Amazon_SageMaker_FAQs/Amazon_SageMaker_FAQs.csv\"\n",
```
Comment (tchattha) [inline 04_Chatbot/00_Chatbot_Claude.ipynb:514]: We cannot keep the downloaded file as part of the repo hence downloading, can you please revert this change and download the file from source.

File: 04_Chatbot/rag_data/Amazon_SageMaker_FAQs.csv
Comment (tchattha) [inline 04_Chatbot/rag_data/Amazon_SageMaker_FAQs.csv:1]: This file can't be part of the repo, please remove.

**Review by tchattha — COMMENTED** [review]
Kindly fix the issues for the merge to complete.

File: 04_Chatbot/rag_data/Amazon_SageMaker_FAQs.csv
Comment (mikejgillespie) [reply 04_Chatbot/rag_data/Amazon_SageMaker_FAQs.csv:1]: FYI: I removed this because the file was giving me a permissions error when I tried to download it. It worked ok if I downloaded it from my Amazon laptop, but failed when I ran it in a Sagemaker Notebook.

Comment (tchattha) [reply 04_Chatbot/rag_data/Amazon_SageMaker_FAQs.csv:1]: we can't have it as part of the repo, we don't have permission to do so.
Please remove the file from this PR and we can approve.

**Review by tchattha — APPROVED** [review]
